src/AccessibilityInsights/MainWindow.xaml

```diff
@@ -345,15 +345,21 @@
                                                     <DataTrigger Binding="{Binding Path=vmHilighter.State, Mode=OneWay, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=local:MainWindow}}" Value="Off">
                                                         <Setter Property="GlyphName" Value="UnSetColor"/>
                                                     </DataTrigger>
+                                                    <Trigger Property="IsMouseOver" Value="True">
```

Comment:
@jalkire, I'd  much prefer to use the `hoverAwareFabricIcon` style here (instead of duplicating the triggers), but I wasn't able to find a neat way to get both the icon binding and the color binding. I'd welcome any suggestions you may have on this

Reply:
You could try setting the [`BasedOn` property](https://docs.microsoft.com/en-us/dotnet/api/system.windows.style.basedon?view=netframework-4.8) of this style up on line 340 to point at `hoverAwareFabricIcon`. Unfortunately XAML's inheritance isn't super flexible so you might end up losing the parent triggers if you set new ones here, but hopefully not